ts/src/bybit.ts

```diff
@@ -3565,7 +3578,12 @@ export default class bybit extends Exchange {
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
          */
         await this.loadMarkets ();
-        const market = this.market (symbol);
+        let market = this.market (symbol);
+        if (market['option']) {
```

Comment:
@Dan-krm no we can't do that, that will introduce a huge unexpected delay upon placing the order.

Reply:
Okay, I also hard coded the amount precision and price precision based on the base currency in createExpiredOption which prevents the issue of having undefined values.

I adjusted the conditional to only set loadAllOptions to true if the values are also undefined or we can remove that all together